frontend/src/pages/TrainingReports/components/SessionCard.js

```diff
@@ -0,0 +1,132 @@
+  session,
+  expanded,
+}) {
+  const { user } = useContext(UserContext);
+
+  const {
+    sessionName,
+    startDate,
+    endDate,
+    objective,
+    objectiveSupportType,
+    objectiveTopics,
+    objectiveTrainers,
+    status,
+    regionId,
+  } = session.data;
+
+  const hasEditPermissions = canEditOrCreateSessionReports(user, parseInt(regionId, DECIMAL_BASE));
+
+  const getSessionDisplayStatusText = () => {
+    if (status && (status === 'In progress' || status === 'Complete')) {
+      return status;
+    }
+    return 'Needs status';
+  };
+
+  const displaySessionStatus = getSessionDisplayStatusText();
+
+  const getSessionStatusIcon = (() => {
+    if (displaySessionStatus === 'In progress') {
+      return <InProgress />;
+    } if (displaySessionStatus === 'Complete') {
+      return <Closed />;
+    }
+    return <NoStatus />;
+  })();
+
+  return (
+    <ul className="ttahub-session-card__session-list usa-list usa-list--unstyled padding-2 margin-top-2 bg-base-lightest radius-lg" hidden={!expanded}>
+      <li className="display-flex padding-bottom-05 flex-align-start">
+        <span className="margin-right-3 minw-15">Session name </span>
+        <div>
+          <span className="margin-right-2">
+            {sessionName}
+          </span>
+          {
+            hasEditPermissions
+              ? (
+                <span>
+                  <Link key={`edit-session-key-${session.id}`} to={`/training-report/${eventId}/session/${session.id}/session-summary`}>
+                    <Pencil />
+                    Edit session
+                  </Link>
+                </span>
+              )
+              : null
+      }
+        </div>
+      </li>
+      <li className="display-flex padding-bottom-05 flex-align-start">
+        <span className="margin-right-3 minw-15">Session dates </span>
+        {`${startDate || ''} - ${endDate || ''}`}
+      </li>
+      <li className="display-flex padding-bottom-05 flex-align-start">
+        <span className="margin-right-3 minw-15">Session objective </span>
+        {objective}
+      </li>
+
+      <li className="display-flex padding-bottom-05 flex-align-start">
+        <span className="margin-right-3 minw-15">Support type </span>
+        {objectiveSupportType}
+      </li>
+
+      <li className="display-flex padding-bottom-05 flex-align-start">
+        <span className="margin-right-3 minw-15">Topics </span>
+        {objectiveTopics && objectiveTopics.length > 0 ? objectiveTopics.join(', ') : ''}
+      </li>
+
+      <li className="display-flex padding-bottom-05 flex-align-start">
+        <span className="margin-right-3 minw-15">Trainers </span>
+        {objectiveTrainers && objectiveTrainers.length > 0 ? objectiveTrainers.join(', ') : ''}
+      </li>
+
+      <li className="display-flex padding-bottom-05 flex-align-start">
+        <span className="margin-right-3 minw-15">Status </span>
+        {getSessionStatusIcon}
+        {displaySessionStatus}
+      </li>

```

Comment:
minor: these list items could be simplified to something like:

```typescript
function SessionListItem({ label, value }) {
  return (
    <li className="display-flex padding-bottom-05 flex-align-start">
      <span className="margin-right-3 minw-15">{label}</span>
      {value}
    </li>
  );
}

// in sessioncard:
 
<SessionListItem label="Session dates" value={`${startDate || ''} - ${endDate || ''}`} />
<SessionListItem label="Session objective" value={objective} />
<SessionListItem label="Support type" value={objectiveSupportType} />
<SessionListItem label="Topics" value={objectiveTopics && objectiveTopics.length > 0 ? objectiveTopics.join(', ') : ''} />
<SessionListItem label="Trainers" value={objectiveTrainers && objectiveTrainers.length > 0 ? objectiveTrainers.join(', ') : ''} />
<SessionListItem label="Status" value={<>{getSessionStatusIcon}{displaySessionStatus}</>} />
```

Reply:
Will keep this as is for now.